views/editor/projects.html

```diff
@@ -26,6 +26,12 @@
           {{ [ "data", "project", key ] | join("-") }}="{{ value }}"
         {% endfor %}
         >
+          <td>
+            <div class="project-favorite">
```

Comment:
I was following the format of how the delete button was put in the table 
```html
<td>
    <div class="project-delete" title="{{ gettext("prjListDeleteProjectBtnTitle") }}">
        <div class="icon-garbage-can">
            <div class="icon-garbage-lid"></div>
        </div>
        {{ gettext("prjListDeleteProjectBtn") }}
    </div>
</td>
```
but you are right, it works just fine if i remove one of the divs.